SimG4CMS/HGCalTestBeam/plugins/AHCalSD.cc

```diff
@@ -88,10 +88,10 @@ uint32_t AHCalSD::setDetUnitId(const G4Step* aStep) {
   const G4VTouchable* touch = preStepPoint->GetTouchable();
 
   int depth = (touch->GetReplicaNumber(1));
-  int incol = ((touch->GetReplicaNumber(0)) % 10);
-  int inrow = ((touch->GetReplicaNumber(0)) / 10) % 10;
-  int jncol = ((touch->GetReplicaNumber(0)) / 100) % 10;
-  int jnrow = ((touch->GetReplicaNumber(0)) / 1000) % 10;
+  int incol = ((touch->GetReplicaNumber(0)) % 100);
+  int inrow = ((touch->GetReplicaNumber(0)) / 100) % 100;
+  int jncol = ((touch->GetReplicaNumber(0)) / 10000) % 10;
+  int jnrow = ((touch->GetReplicaNumber(0)) / 100000) % 10;
```

Comment:
These factors of 10 are magic numbers. The values in `DDAHcalModuleAlgo.cc ` must match those in `AHCalSD.cc` for this code to work right. The best way to ensure that is for both files to use the same constants. Since this PR is bug fix, this issue doesn't need to be fixed now, but it would be good to clean it up eventually.

Reply:
@cvuosalo I agree, on the other side this was the situation even before, I believe the adjustment using named constants can be factorized